core/src/components/select/select.md.scss

```diff
@@ -68,9 +68,27 @@
   transform: scale(1);
 }
 
+/**
+ * Adjust the highlight up by 1px
+ * so it is not cut off by the
+ * the item's line (if one is present).
+ */
+:host(.in-item) .select-highlight {
+  @include position(null, null, 0, 0);
+}
```

Comment:
Noticed that the highlight doesn't stretch across the entire width of the item (note the uncovered section of the item's border at the bottom right); should it?
![item](https://i.gyazo.com/6ec68fb550f2ef2b21ee3816d77afc99.png)

Reply:
Yes/No. The item has end padding which is going to shift the select over by some number of pixels. We'll want to find a way to account for this, but I don't think it's a blocker at the moment.